src/app/components/Article/index.jsx

```diff
@@ -20,17 +18,17 @@ const Article = ({ lang, title, blocks, id }) => {
         <link rel="canonical" href={canonicalLink} />
       </Helmet>
       <Header />
-      <MainContent blocks={blocks} />
+      {children}
       <Footer />
     </Fragment>
   );
 };
 
 Article.propTypes = {
+  children: node.isRequired,
```

Comment:
More of a query than a review point - is it possible to ensure this is at least 2 nodes?
NB: We know a valid article must have a headline and a paragraph hence 2 nodes.